scripts/convert_hf_checkpoint.py

```diff
@@ -17,21 +17,19 @@
 @torch.no_grad()
 def convert_hf_checkpoint(
     *,
-    output_dir: Path = Path("checkpoints/lit-llama"),
-    ckpt_dir: Path = Path("checkpoints/hf-llama/"),
+    output_dir: Path = Path("checkpoints/lit-llama/7B"),
+    checkpoint_dir: Path = Path("checkpoints/hf-llama/7B"),
     model_size: str = "7B",
```

Comment:
Do the new weights support other models than 7B, if not, maybe adding "not supported yet" when other sizes are used

Reply:
We could add an internal check on weights sizes in a future PR

Reply:
ok happy to do that later